CHANGELOG.md

```diff
@@ -1,34 +1,181 @@
-# Release 9.0.1
+# Release 10.0.0
 
 **NOTE:**
-_This changelog only contains release notes for PWA Studio and Venia 9.0.1._
+_This changelog only contains release notes for PWA Studio and Venia 10.0.0._
 _For older release notes, see [PWA Studio releases][]._
 
 ## Table of contents
 
--   [What's new in 9.0.1](#whats-new-in-901)
+-   [What's new in 10.0.0](#whats-new-in-1000)
 -   [Pull requests merged in this release](#pull-requests-merged-in-this-release)
 -   [Known issues](#known-issues)
 -   [Upgrading from a previous version](#upgrading-from-a-previous-version)
 
-## What's new in 9.0.1
+## What's new in 10.0.0
 
-PWA Studio 9.0.1 contains the same features, refactors, and various improvements in the previous [9.0.0 release][] plus a hotfix for the `create-pwa` CLI tool.
+PWA Studio 10.0.0 contains new features, refactors, bug fixes, and various improvements.
 
-[9.0.0 release]: https://github.com/magento/pwa-studio/releases/tag/v9.0.0
+### Build report tool
 
-### Hotfix for `create-pwa`
+The build report tool is a Buildpack CLI command that returns information about their storefront setup and development environment.
+This feature makes it easier to provide information when reporting issues or for general debugging purposes.
 
-This release fixes a bug that crashes the scaffolding tool whenever you run `yarn create @magento/pwa` or `npx @magento/create-pwa`.
+The following command is now available for newly scaffolded projects:
+
+```sh
+yarn build:report
+```
+
+[![Image from Gyazo](https://i.gyazo.com/960b390007953778d67d298bd27b4886.png)](https://gyazo.com/960b390007953778d67d298bd27b4886)
+
+### Check or Money Order payments
+
+A PWA Studio extension that allows check or money order payments is now available thanks to the contribution of community member [Lars Roettig][].
+If the Magento backend has the **Check or Money Order** option enabled, this extension lets storefront customers use this payment option.
```

Comment:
`Make check Payable to` and `Send Check to`  does not have GraphQL coverage, should this be noted in known issues or somewhere?  More info https://github.com/magento/pwa-studio/pull/2969#issuecomment-797654096